app/models/server/models/Statistics.js

```diff
@@ -4,7 +4,7 @@ export class Statistics extends Base {
 	constructor() {
 		super('statistics');
 
-		this.tryEnsureIndex({ createdAt: 1 });
+		this.tryEnsureIndex({ createdAt: -1 });
```

Comment:
the only find using this index was [findLast](https://github.com/RocketChat/Rocket.Chat/pull/20832/files#diff-aa52c27dda94556a7ffd6f5636e221f2c9107b6ef07b33fc9c2c329b65adeffbR16) and it sorts on reverse order